ua-snap/epa-justice
# Fix CI math for aggregated communities

This PR closes #16 

In order to get correct low and high confidence interval values for communities with aggregated data (right now just JBER and Eagle River), pooled standard deviations were back-calculated from the unaggregated data and used to calculate the pooled confidence interval values. There is an intermediate step (and function) to calculate the adult population variance, which was necessary to get the pooled standard deviation.

A new `qc/` directory was also added, and the CSVs in this directory were shared with Nancy and Micah to double check the aggregation math. So for the purposes of this PR, test to make sure the code runs as expected but don't worry too much about the math - if both Nancy and Micah approve of the results, I'd say we are good to go!

This code could be cleaner, but I found it helpful in my troubleshooting to really spell out all the variables and document the formulas, etc. Hopefully it's not too hard to follow, let me know if you see anything weird, redundant, or that seems totally unnecessary and we can lint.

To test:

- Run the both of the notebooks and confirm that there are no changes introduced to the results tables or the shapefile.
